CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## [0.15.0](https://github.com/eclipse/lemminx/milestone/20?closed=1) (February 2, 2021)
+
+### Enhancements
+ * Generate a native binary using GraalVM. See [#860](https://github.com/eclipse/lemminx/pull/860).
+ * Indicate if the server is a binary in the startup message. See [#949](https://github.com/eclipse/lemminx/issues/949).
+ * Allow LemMinX extensions to contribute to `WorkspaceService`. See [#966](https://github.com/eclipse/lemminx/pull/966).
+ * Add new formatting setting `xml.format.splitAttributesIndentSize`. See [#952](https://github.com/eclipse/lemminx/pull/952).
+ * Disable XSD validation when `xsi:schemaLocation` doesn't declare the hint for the document element root. See [#953](https://github.com/eclipse/lemminx/pull/953).
+ * Manage namespaces / prefix validation with a setting. See [#960](https://github.com/eclipse/lemminx/issues/960).
+
+### Bug Fixes
+ * Avoid trailing space in processing instructions. See [redhat-developer/vscode-xml#934](https://github.com/redhat-developer/vscode-xml/issues/372).
```

Comment:
redhat-developer/vscode-xml#372 was fixed by eclipse/lemminx#934 . I guess in this case you would refer to the issue on the vscode-xml side ? Either way, mixing the links seemed a bit weird.